plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
```

Comment:
I'm not really happy with evaluating the scripts early. That limits their use quite a bit since we lose all kinds of runtime evaluations. Or am I missing something?